app/views/components/primary_navigation.scala.html

```diff
@@ -60,15 +60,13 @@
             </nav>
         </div>
         <div class="moj-primary-navigation__search">
-            <div class="moj-search moj-search--ondark moj-search--inline"><form action="/205-classification/search/advanced-search-results" method="get">
+            <div class="moj-search moj-search--ondark moj-search--inline">
+                @helper.form(action = controllers.routes.SearchController.search(), 'novalidate -> "novalidate") {
```

Comment:
Should we be using uk.gov.hmrc.play.views.html.helpers.form?
I can't recall if play's helper.form includes the CSRF token or if that was version dependent.

Reply:
It includes the CSRF token. We have replaced it with the uk.gov.hmrc.play.views.html.helpers.form 👍